packages/docs-reanimated/docs/layout-animations/layout-transitions.mdx

```diff
@@ -4,195 +4,376 @@ sidebar_position: 3
 
-```js
-    // AnimatedComponent - component created by createAnimatedComponent or imported from Reanimated
-    <AnimatedComponent layout={Transition} >
+#### Easing <Optional/>
+
+Time-based modifiers relay on [`withTiming`](/docs/animations/withTiming) function.
+
+```javascript
+LinearTransition.easing(Easing.ease);
 ```
 
-### 3. Customize the animation
+- `.easing(easingFunction: EasingFunction)` is an easing function which defines the animation curve. Defaults to `Easing.inOut(Easing.quad)`
+
+:::note
+The `.easing( ... )` modifier have no effect when `.springify()` is used.
+:::
+
+#### Spring-based <Optional/>
+
+Spring-based modifiers relay on [`withSpring`](/docs/animations/withSpring) function.
+Just as in withSpring config, spring-based modifiers can be physics-based (has `mass` and `damping` modifiers) or duration-based (has `duration` and `dampingRatio` modifiers). Usage of any time-based spring modifiers erases the effect of any physics-based spring modifier. Defaults to physics-based.
+
+```javascript
+LinearTransition.springify()
+  .damping(30)
+  .mass(5)
+  .rotate(20)
+  .stiffness(10)
+  .overshootClamping(false)
+  .restDisplacementThreshold(0.1)
+  .restSpeedThreshold(5);
+```
+
+- `.springify()` enables the spring-based animation configuration.
+- `.damping(value: number)` decides how quickly a spring stops moving. Higher damping means the spring will come to rest faster. Defaults to `10`.
+- `.dampingRatio(value: number)` decides how damped the spring is. Value `1` means the spring is critically dmaped, and value `>1` means the spring is overdamped. Defaults to `0.5`.
+- `.mass(value: number)` is the weight of the spring. Reducing this value makes the animation faster. Defaults to `1`.
+- `.rotate(degree: string)` lets you rotate the element.
+- `.stiffness(value: number)` decides how bouncy the spring is. Defaults to `100`.
+- `.overshootClamping(value: boolean)` decides whether a spring can bounce over the designated position. Defaults to `false`.
+- `.restDisplacementThreshold(value: number)` is the displacement below which the spring will snap to the designated position without further oscillations. Defaults to `0.001`.
+- `.restSpeedThreshold(value: number)` is the speed in pixels per second from which the spring will snap to the designated position without further oscillations. Defaults to `2`.
+
+#### Common <Optional/>
+
+```javascript
+LinearTransition.delay(500)
+  .reduceMotion(ReduceMotion.Never)
+  .withInitialValues({ transform: [{ translateY: 420 }] })
+  .withCallback((finished) => {
+    console.log(`finished without interruptions: ${finished}`);
+  });
+```
+
+- `.duration(duration: number)` sets length of the animation (in milliseconds). Defaults to '300'.
+- `.delay(durationMs: number)` is the delay before the animation starts (in milliseconds). Defaults to `0`.
+- `.reduceMotion(reduceMotion: ReduceMotion)` determines how the animation responds to the device's reduced motion accessibility setting.
```

Comment:
Maybe it would be helpful if we link pages that describes type `ReduceMotion` here